gke_deployment/slackbot-service.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
```

Comment:
In my optionion, resource(e.g. Service, Deployment...) manifests is put `manifests` dir.

The platform should be decided by the users.